### DISCUSSION TITLE
The positions of my function are duplicated!!! - coc-references

### DISCUSSION THREAD

**Discussion by bty102** [neoclide/coc.nvim#4625] [General]
I use (coc-references) command. But the positions of my function are duplicated. What should I do?

![image](https://user-images.githubusercontent.com/125985488/235608732-49ed9575-3e76-405b-bead-3c77d139dc6a.png)

**Comment by fannheyward** [comment]
Checkout `:CocList services`, looks like you've two clangd setup.

**Reply by bty102** [reply]
![image](https://user-images.githubusercontent.com/125985488/236174203-402510ec-3e98-46d4-bb53-0ec8f544bb06.png)
This is the result when I use the CocList services command.

**Reply by bty102** [reply]
I see only one clangd is running.

**Reply by fannheyward** [reply]
Have no idea about this, coc.nvim is a LSP client, the server, here's clangd, returns references results, client displayed them. Check language server results https://github.com/neoclide/coc.nvim/wiki/Debug-language-server#using-output-channel

**Reply by bty102** [reply]
Thank you for ur feedback

**Reply by bty102** [reply]
I think the issue is coming from my language server.
